packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -131,6 +133,35 @@ export class ChainService implements ChainServiceInterface {
     );
   }
 
+  public async handleChainRequests(
```

Comment:
This new method is used by the wallet to deal with `ChainRequest`s from the `engine`.